tools/runner

```diff
@@ -173,14 +173,14 @@ if test_params['mode'] is None:
 
 for key in libs.keys():
     if key in test_params['tags']:
-        test_params['files'] += [
+        test_params['files'] = [
             os.path.abspath(os.path.join(dirs['third_party'], p))
             for p in libs[key]['files']
-        ]
-        test_params['incdirs'] += [
+        ] + test_params['files']
```

Comment:
A simpler way to extend the list from the beginning without having to type `test_params['files']` twice would be:
```
test_params['files'][0:0] = [<your_comprehension>]
```
But your solution is fine too